extensions/amp-bind/0.1/validator-amp-bind.protoascii

```diff
@@ -46,39 +46,20 @@ tags: {  # <amp-state> (json)
   }
   spec_url: "https://www.ampproject.org/docs/reference/components/amp-bind"
 }
-tags: {  # <amp-state> with json child
```

Comment:
@Gregable Since I'm out the next two days, could you take a look at this and recommend a way forward? I don't think we want to rely on ordering the tagspecs in the protoascii and I'm not thinking of anything immediately obvious to apply these rules.